src/commands/ssh.ts

```diff
@@ -163,6 +169,27 @@ const ssh = async (args: yargs.ArgumentsCamelCase<SshCommandArgs>) => {
   // Prefix is required because the backend uses it to determine that this is an AWS request
   const authn = await authenticate();
   await validateSshInstall(authn);
+
+  if (args.requestGroup) {
+    // When requesting for a group we need 2 requests. The first request grants access to the group
+    // requests will detect that the node is a part of the group and can use the group permission
+    // to gain access to the node.
+    const response = await request(
+      {
+        ...pick(args, "$0", "_"),
+        arguments: ["ssh", "group", "--destination", args.destination],
```

Comment:
how does this work with pre-existing access?

Reply:
There's a different bug for groups right now where making a request to a group with pre-existing access creates a duplicate request. I made a separate  PR to fix it independently here: https://github.com/p0-security/app/pull/1569

Without the fix we'll keep requesting new access requests for a group of nodes. With the fix we'll be able to gain access without making another request.

Essentially these two scenarios will work without generating any extra requests:
1. `p0 request ssh group Miguel` and then `p0 ssh --request-group private-node`
2. `p0 ssh --request-group private-node` and `p0 ssh --request-group private-node`